settings.gradle

```diff
@@ -56,6 +56,7 @@ include ':libs:analytics'
 include ':libs:editor'
 include ':libs:processors'
 include ':libs:annotations'
+include ':libs:gravatar'
```

Comment:
⚠️ We don't need this. Gravatar is not in `libs` folder

Reply:
Removed in [50aea71](https://github.com/wordpress-mobile/WordPress-Android/pull/20590/commits/50aea7192235bc25e2a31a9e37a2a1b52002107a)